api/hypershift/v1beta1/hostedcluster_types.go

```diff
@@ -1248,9 +1248,72 @@ const (
 // +kubebuilder:validation:Enum=Karpenter
 type Provisioner string
 
+// Configures when and how to scale down cluster nodes.
+type ScaleDownConfig struct {
+	// enabled determines whether the Cluster Autoscaler should scaled down this cluster.
+	Enabled bool `json:"enabled"`
+
+	// delayAfterAdd sets how long after scale up the scale down evaluation resumes
+	// +kubebuilder:validation:Pattern=([0-9]*(\.[0-9]*)?[a-z]+)+
```

Comment:
Are you trying to validate that this is a duration? 

Since validation for durations is complex to implement, it's preferred to include a unit and turn this into an int32, eg, `delayAfterAddSeconds` and then it's unambiguous of the unit, the value is clear to validate and it's easy to validate whether you're in go, or not